cpp/src/groupby/hash/groupby.cu

```diff
@@ -68,12 +68,18 @@ namespace {
 
 // TODO: replace it with `cuco::static_map`
 // https://github.com/rapidsai/cudf/issues/10401
-using map_type = concurrent_unordered_map<
-  cudf::size_type,
-  cudf::size_type,
-  cudf::experimental::row::hash::device_row_hasher<cudf::detail::default_hash,
-                                                   cudf::nullate::DYNAMIC>,
-  cudf::experimental::row::equality::device_row_comparator<cudf::nullate::DYNAMIC>>;
+template <typename ComparatorType>
+using map_type =
+  concurrent_unordered_map<cudf::size_type,
+                           cudf::size_type,
+                           cudf::experimental::row::hash::
+                             device_row_hasher<cudf::detail::default_hash, cudf::nullate::DYNAMIC>,
+                           ComparatorType>;
+
+template <bool has_nested_columns>
+using comparator_type =
+  cudf::experimental::row::equality::device_row_comparator<has_nested_columns,
+                                                           cudf::nullate::DYNAMIC>;
```

Comment:
Where is this being used?

Reply:
It was vestigial. Didn't realize my refactor eliminated the need for it.